packages/react/ds/src/utils.ts

```diff
@@ -0,0 +1,13 @@
+export function slugify(value?: string) {
```

Comment:
Sorry I didn't catch why would we use slugify? 😄
Can we add few bits for this file as a documentation.

Reply:
It's used because the `value` is used in the `id` field of the tab.
For avoid spacing or chars not valid in `id` field I opted for using that slug function.

Reply:
Thanks! Can we please add documentation for this file maybe with comments on top?